packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
I don't think it was/is needed, because we always distribute a configure in tarballs of releases.